<h3>Lab Sheet 18 Solutions</h3>

<h4>Lab Sheet 18 Problem 1</h4>

Find the linear equation for the plane in
three-dimensional space that contains
$$  P=(1,5,1)\hspace{0.5in}Q=\left(-1,-\frac{1}{2},0\right)\hspace{0.5in}R=(-4,7,3)\notag
$$

<img src="./diagrams/gc-lab-18-partial-solutions-01-1of1.png" width="70%" height="70%">

<h4>Lab Sheet 18 Problem 2</h4>

Calculate the angle in degrees, minutes, and seconds
between the following two vectors
$$  \vec{v}=\left(
    \begin{array}{c}
      2 \\
      -2 \\
      -3
    \end{array}\right)\mbox{ and }\vec{w}=7\vec{i}-\vec{j}-2\vec{k}\notag
$$

<img src="./diagrams/gc-lab-18-partial-solutions-02-1of1.png" width="70%" height="70%">

<h4>Lab Sheet 18 Problem 3</h4>

What is the Maclaurin series expansion for $(1-x)^{-1}$?

$$f^{(n)}(x)=\frac{n!}{(1-x)^{n+1}}$$

$$
\frac{1}{1-x}=\sum_{n=0}^{\infty}x^{n}
$$

Where does this expansion converge?

For $|x|<1$.

<h4>Lab Sheet 18 Problem 4</h4>

John wants to get to the bus stop, the bus stop is across a grassy
park, 2000 feet west and 600 feet north of his starting position. John
can walk west along the edge of the park on the sidewalk at a speed of
6 ft/sec and 4 ft/sec through the grass. How far should he walk on the
sidewalk before veering off onto the grass if he wishes to get to the
bus stop as quickly as possible?

$$f(x,y)=\frac{x}{6}+\frac{y}{4}$$

$$g(x,y)=y^{2}-600^2-(2000-x)^{2}$$

In [1]:
y=var('y');w=var('w');l=var('l')

In [2]:
g(x,y)=y^2-600^2-(2000-x)^2

In [3]:
g.expand()

(x, y) |--> -x^2 + y^2 + 4000*x - 4360000

$$L(x,y,\lambda)=\frac{x}{6}+\frac{y}{4}-\lambda(y^{2}-x^{2}+4000x-4360000)$$

$$\frac{\partial{}L}{\partial{}x}=\frac{1}{6}+2\lambda{}x-4000\lambda=0$$

$$\frac{\partial{}L}{\partial{}y}=\frac{1}{4}-2\lambda{}y=0$$

$$\frac{\partial{}L}{\partial{}\lambda}=y^{2}-x^{2}+4000x-4360000=0$$

In [4]:
A=solve([1/6+2*l*x-4000*l==0,1/4-2*l*y==0,y^2-x^2+4000*x-4360000],x,y,l,solution_dict=True)

In [5]:
A

[{x: -240*sqrt(5) + 2000, y: 360*sqrt(5), l: 1/14400*sqrt(5)},
 {x: 240*sqrt(5) + 2000, y: -360*sqrt(5), l: -1/14400*sqrt(5)}]

In [6]:
n(A[0][x])

1463.34368540005

In [7]:
n(A[0][y])

804.984471899924

John should walk $1463$ feet on the sidewalk and then walk across the grass.

<h4>Lab Sheet 18 Problem 5</h4>

Given the function $f(x,y,z)=\sin(x)\cos(y)e^{z}$, what are the corresponding
partial derivatives $f_{x},f_{y}$ and $f_{z}$?

$$f_{x}=\cos(x)\cos(y)e^{z}\qquad{}f_{y}=-\sin(x)\sin(y)e^{z}\qquad{}f_{z}=\sin(x)\cos(y)e^{z}$$

For the same function $f(x,y,z)$, what is the corresponding Laplacian?

$$\Delta{}f=\nabla\cdot\nabla{}f=\sum_{i=1}^{n}\frac{\partial^{2}f}{\partial{}x^{2}_{i}}=-\sin(x)\cos(y)e^{z}$$

<h4>Lab Sheet 18 Problem 6</h4>

Find the maximum and minimum values of
$$  f(x,y)=81x^{2}+y^{2}\notag
$$
subject to the constraint $4x^{2}+y^{2}=9.$

<img src="./diagrams/gc-lab-18-partial-solutions-06-1of6.png" width="120%" height="120%">

<img src="./diagrams/gc-lab-18-partial-solutions-06-2of6.png" width="120%" height="120%">

<img src="./diagrams/gc-lab-18-partial-solutions-06-3of6.png" width="120%" height="120%">

<img src="./diagrams/gc-lab-18-partial-solutions-06-4of6.png" width="120%" height="120%">

<img src="./diagrams/gc-lab-18-partial-solutions-06-5of6.png" width="120%" height="120%">

<img src="./diagrams/gc-lab-18-partial-solutions-06-6of6.png" width="120%" height="120%">

<h4>Lab Sheet 18 Problem 7</h4>

Find the maximum and minimum values of
$$  f(x,y,z)=3x^{2}+y\notag
$$
subject to the constraints $4x-3y=9$ and $x^{2}+z^{2}=9$.

<img src="./diagrams/gc-lab-18-partial-solutions-07-1of6.png" width="120%" height="120%">

<img src="./diagrams/gc-lab-18-partial-solutions-07-2of6.png" width="75%" height="75%">

<img src="./diagrams/gc-lab-18-partial-solutions-07-3of6.png" width="120%" height="120%">

<img src="./diagrams/gc-lab-18-partial-solutions-07-4of6.png" width="120%" height="120%">

<img src="./diagrams/gc-lab-18-partial-solutions-07-5of6.png" width="120%" height="120%">

<img src="./diagrams/gc-lab-18-partial-solutions-07-6of6.png" width="120%" height="120%">

<h4>Lab Sheet 18 Problem 8</h4>

Consider an open box with no top. The box has volume $32$
(cubic units) and dimensions $x,y,z$. Using Lagrange multipliers, find
the dimensions of the box with minimal surface area.

<img src="./diagrams/openbox.png" width="20%" height="20%">

<img src="./diagrams/gc-lab-18-partial-solutions-08-1of3.png" width="60%" height="60%">

<img src="./diagrams/gc-lab-18-partial-solutions-08-2of3.png" width="60%" height="60%">

<img src="./diagrams/gc-lab-18-partial-solutions-08-3of3.png" width="60%" height="60%">

<h4>Lab Sheet 18 Problem 9</h4>

Suppose $x+y+z=A$. What values of $x,y,z$ are such that
$x^{2}+y^{2}+z^{2}$ is the smallest?

<img src="./diagrams/gc-lab-18-partial-solutions-09-1of2.png" width="60%" height="60%">

<img src="./diagrams/gc-lab-18-partial-solutions-09-2of2.png" width="60%" height="60%">